src/platforms/dart/configuration/integrations/dio.mdx

```diff
@@ -66,12 +81,11 @@ Capturing transactions requires that you first <PlatformLink to="/performance/">
 </Note>
 
 The Dio integration starts a span out of the active span bound to the scope for each HTTP request.
-This is currently enabled by default. The following example shows how to disable it.
 
 ```dart
 import 'package:sentry_dio/sentry_dio.dart';
 
 final dio = Dio();
 
-dio.addSentry(networkTracing: false);
+dio.addSentry();
```

Comment:
I was wondering why not keep the info on how to disable it, but I guess you want to have examples in the docs that can be copy pasted to enable it?

Reply:
Because there's a note above telling that this is only possible if performance is enabled, otherwise it's clearly disabled.